deploy/cr.yaml

```diff
@@ -17,6 +17,10 @@ spec:
 #  tls:
 #    # 90 days in hours
 #    certValidityDuration: 2160h
+#    issuerConf:
```

Comment:
@inelpandzic maybe we need to have e2e test for it. We have it in PXC. Just copy a logic. @tplavcic what do you think?

Reply:
@hors I think the reason for have it in e2e in PXC is probably because we either couldn't test it easily with unit tests or just didn't consider it. I don't think we need to have it as a e2e test and that way just make our tests even longer. 

Unit test for this case is perfectly good IMO.